<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgMotionModeParameters</class>
 <widget class="QDialog" name="CQDlgMotionModeParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Joint Motion Mode Parameters</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QCheckBox" name="qqExplicitHandling">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Explicit handling</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqInfiniteAcceleration">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Infinite acceleration</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqInvert">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Invert target velocity at limits</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Target velocity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqTargetVelocity">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqAcceleration">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Acceleration</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqClose">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqExplicitHandling</tabstop>
  <tabstop>qqInfiniteAcceleration</tabstop>
  <tabstop>qqInvert</tabstop>
  <tabstop>qqTargetVelocity</tabstop>
  <tabstop>qqAcceleration</tabstop>
  <tabstop>qqClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
